baybe/surrogates/base.py

```diff
@@ -83,6 +86,34 @@ def to_botorch(self) -> Model:
 
         return AdapterModel(self)
 
+    @staticmethod
+    def _get_parameter_scaler(
+        parameter: Parameter,
+    ) -> ScalerProtocol | Literal["passthrough"]:  # noqa: F821
+        """Return the scaling method to be used for the given parameter."""
+        return MinMaxScaler()
+
+    def _make_input_scaler(
+        self, searchspace: SearchSpace, measurements: pd.DataFrame
```

Comment:
`measurements` is not (yet?) used here, I guess this is related to the `TODO` below?

Reply:
Yes exactly 👍🏼 Will close here since duplicate